src/gc/impl/conservative/gc.d

```diff
@@ -2945,6 +3152,77 @@ struct Pool
             }
         }
     }
+
+    pragma(inline,true)
+    void setPointerBitmap(void* p, size_t s, size_t allocSize, uint attr, const TypeInfo ti) nothrow
+    {
+        if (!(attr & BlkAttr.NO_SCAN))
+            setPointerBitmap(p, s, allocSize, ti, attr);
+    }
+
+    pragma(inline,false)
+    void setPointerBitmap(void* p, size_t s, size_t allocSize, const TypeInfo ti, uint attr) nothrow
```

Comment:
It may be clearer during reading and less prone to mistakes by future maintainers if these two functions were distinguished by having different names rather than just having the order of their last two arguments swapped.